lib/api/wsl.dart

```diff
@@ -200,11 +201,19 @@ class WSLApi {
         mode: ProcessStartMode.normal, runInShell: true);
   }
 
-  /// Start Windows Terminal
+  /// Start Windows Terminal\
+  /// *(falls back to PowerShell on exception)*
   void startWindowsTerminal(String distribution) async {
-    List<String> args = ['wt', 'wsl', '-d', distribution, '--cd', '~'];
-    Process.start('start', args,
-        mode: ProcessStartMode.normal, runInShell: true);
+    List<String> launchWslHome = ['wsl', '-d', distribution, '--cd', '~'];
+    try {
+      await Process.start('wt', launchWslHome);
```

Comment:
Had to remove `runInShell` otherwise exception is not provided to our process. I don't know if this has other unwanted side effects, but it does launch `wt` correctly if it is installed. Please confirm.